test_mean.py

```diff
@@ -0,0 +1,34 @@
+from mean import *
+
+def test_ints():
+    num_list = [1,2,3,4,5]
+    obs = mean(num_list)
+    exp = 3
+    assert obs == exp
+
+def test_zero():
+    num_list=[0,2,4,6]
+    obs = mean(num_list)
+    exp = 3
+    assert obs == exp
+
+def test_double():
```

Comment:
Please include a docstring describing what each test function does and what it testing.